frontend/src/index.css

```
@@ -40,3 +40,91 @@ body {
   border-right: 4px solid #f43f5e;
   background: #fff1f2;
 }
+
+/* Custom zoom slider — track fill follows --range-progress (0-100),
+   set inline so the filled portion tracks the thumb in every browser. */
+.zoom-slider {
+  --range-progress: 0%;
+  -webkit-appearance: none;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)